resolwe_bio/processes/reads_processing/cutadapt.yml

```diff
@@ -400,78 +410,55 @@
+      down_primers_fwd='{% for primer in adapters.down_primers_seq_fwd %}-a {{primer}} {% endfor %}'
       down_primers_fwd=${down_primers_fwd%?}
-      down_primers_rev='{% for primer in down_primers_seq_rev %}-a {{primer}} {% endfor %}'
+      down_primers_rev='{% for primer in adapters.down_primers_seq_rev %}-A {{primer}} {% endfor %}'
       down_primers_rev=${down_primers_rev%?}
-      up_primers_fwd='{% for primer in up_primers_seq_fwd %}-g {{primer}} {% endfor %}'
+      up_primers_fwd='{% for primer in adapters.up_primers_seq_fwd %}-g {{primer}} {% endfor %}'
       up_primers_fwd=${up_primers_fwd%?}
-      up_primers_rev='{% for primer in up_primers_seq_rev %}-g {{primer}} {% endfor %}'
+      up_primers_rev='{% for primer in adapters.up_primers_seq_rev %}-G {{primer}} {% endfor %}'
       up_primers_rev=${up_primers_rev%?}
 
       # Decompress primer files
-      {% if up_primers_file_fwd %}
-        gunzip -c {{up_primers_file_fwd.fasta.file}} > 5_primers_fwd.fasta
+      {% if adapters.up_primers_file_fwd %}
+        gunzip -c {{adapters.up_primers_file_fwd.fasta.file}} > 5_primers_fwd.fasta
       {% endif %}
-      {% if up_primers_file_rev %}
-        gunzip -c {{up_primers_file_rev.fasta.file}} > 5_primers_rev.fasta
+      {% if adapters.down_primers_file_fwd %}
+        gunzip -c {{adapters.down_primers_file_fwd.fasta.file}} > 3_primers_fwd.fasta
       {% endif %}
-      {% if down_primers_file_fwd %}
-        gunzip -c {{down_primers_file_fwd.fasta.file}} > 3_primers_fwd.fasta
+      {% if adapters.up_primers_file_rev %}
+        gunzip -c {{adapters.up_primers_file_rev.fasta.file}} > 5_primers_rev.fasta
       {% endif %}
-      {% if down_primers_file_rev %}
-        gunzip -c {{down_primers_file_rev.fasta.file}} > 3_primers_rev.fasta
+      {% if adapters.down_primers_file_rev %}
+        gunzip -c {{adapters.down_primers_file_rev.fasta.file}} > 3_primers_rev.fasta
       {% endif %}
 
-      # Run Cutadapt on forward reads
       cutadapt \
-        {% if headcrop_fwd %}-u {{headcrop_fwd}}{% endif %} \
-        {% if crop_fwd %}-u -{{crop_fwd}}{% endif %} \
-        {% if leading %}-q {{leading}}{% endif %} \
-        {% if trailing %}-q {{trailing}},0{% endif %} \
-        {% if polya_tail %}-a A{$polya_tail}N{1}{% endif %} \
-        {% if down_primers_seq_fwd %}$down_primers_fwd{% endif %} \
-        {% if up_primers_seq_fwd %}$up_primers_fwd{% endif %} \
-        {% if down_primers_file_fwd %}-a file:3_primers_fwd.fasta{% endif %} \
-        {% if up_primers_file_fwd %}-g file:5_primers_fwd.fasta{% endif %} \
-        {% if times %}--times {{ times }} {% endif %} \
-        {% if match_read_wildcards %}--match-read-wildcards {% endif %} \
-        {% if error_rate %}--error-rate {{ error_rate }} {% endif %} \
-        -O {{min_overlap_fwd}} \
-        -o "${NAME1}_trimmed.fastq.gz" in_fw_reads.fastq.gz > report.txt
+        {% if filtering.minlen %}-m {{ filtering.minlen }}{% endif %} \
```

Comment:
Usage of short option e.g `-m` vs. `--minimum-length` is inconsitent. I would suggest to always use longer notation because it improves readability.